install/running_install.adoc

```diff
@@ -403,6 +403,38 @@ ifdef::openshift-origin[]
 endif::[]
 ----
 
+[running-the-advanced-installation-openstack]
+==== Running the Installation Playbook for OpenStack
+
+[IMPORTANT]
+====
+The OpenStack installation playbook is a Technology Preview feature.
+ifdef::openshift-enterprise[]
+Technology Preview features are not supported with Red Hat production service
+level agreements (SLAs), might not be functionally complete, and Red Hat does
+not recommend to use them for production. These features provide early access to
+upcoming product features, enabling customers to test functionality and provide
+feedback during the development process.
+
+For more information on Red Hat Technology Preview features support scope, see
+https://access.redhat.com/support/offerings/techpreview/.
+endif::[]
+====
+
+If you want to install {product-title} on an existing OpenStack installation,
+you can do so by using the playbook for OpenStack. For more information about
+the playbook, including detailed prerequisites, see
```

Comment:
@kalexand-rh Can this be tightened up a bit? 
_To install {product-title} on an existing OpenStack installation, use the OpenStack playbook_